cmd/autoconfigbrancher/main.go

```diff
@@ -199,7 +199,7 @@ func main() {
 	run(cmd, args...)
```

Comment:
Will revert https://github.com/openshift/ci-tools/pull/659/files
when everything is working with sanitize tool